### ISSUE TITLE
HTML doc typos

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#838]
**Reported by anonymous on 25 Sep 2012 13:54 UTC**
The HTML documentation of models
```
Modelica.Blocks.Sources.CombiTimeTable
Modelica.Blocks.Sources.TimeTable
Modelica.Electrical.Analog.Sources.TableCurrent
Modelica.Electrical.Analog.Sources.TableVoltage
Modelica.Electrical.Spice3.Sources.I_pwl
Modelica.Electrical.Spice3.Sources.V_pwl
```
needs to be fixed from
```
, in order to not integrate over a discontinuous or not differentiable points.
```
to
```
, in order to not integrate over a discontinuity or not differentiable points.
```
.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/838

**Comment by modelica-trac-importer** [comment]
**Comment by hansolsson on 25 Sep 2012 14:35 UTC**
I don't find the new version to be good either.

Let's start by figuring out what is meant and then describe that instead of patching it. The issue is that the table-output is not continuously differentiable (except when splines are selected in the models that allow it) - but the events ensure that the integrator does not see this.

Thus I would write something like:

"The table is implemented in a numerically sound way by generating time events at interval boundaries, this generates continuously differentiable values for the integrator."

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 23 Jan 2013 08:47 UTC**
FIXED in 7e733dcd4fd4500b8e15597f1d2797d33f3f5c49 using a slight variation of Hans' proposal:
```diff
- , in order to not integrate over a discontinuity or not differentiable points.
+ . This generates continuously differentiable values for the integrator.
```
